WHATSNEW

```diff
@@ -27,6 +27,11 @@ Fixed bugs:
    copy happened to be the same source file (symlinked back
    to itself).
 
+ * Bugzilla Report 58683 - Fixed the issue where symlink creation
```

Comment:
I overlooked this through your earlier changes to WHATSNEW. We use to add the bugzilla issue as last line of the added entry and I think it would be good to keep a common format.

Reply:
Fixed and updated the PR.